src/utils.jl

```diff
@@ -1,14 +1,45 @@
 # Arrays
+"""
+    nfan(n_out, n_in=1) -> Tuple
+    nfan(dims...)
+    nfan(dims::Tuple)
+
+Calculate the maximum numbers of input and output data each neuron weight in the layer connects to.
```

Comment:
This settence is not utterly clear, maybe something like this
```suggestion
For a layer characterized by dimensions  `dims`,  return a tuple `(fan_in, fan_out)`, where `fan_in`
is the number of input neurons connect to an output one, and `fan_out` is the number of output neurons connected to an input one.
```
?

Reply:
Much clearer than my words, thanks